ch01/03_urlify.go

```diff
@@ -55,3 +57,10 @@ func URLify(s string) string {
 
 	return string(runes)
 }
+
+func UrlifyRegex(s string) string {
```

Comment:
```suggestion

// UrlifyRegex returns the s string with HTML-encoded space characters.
func UrlifyRegex(s string) string {
```
It's an exported method; let's add some comments to it.